scijava-common3/pom.xml

```diff
@@ -11,8 +11,8 @@
 
 	<artifactId>scijava-common3</artifactId>
 
-	<name>SciJava Common 3 </name>
```

Comment:
Hmm, we should talk about this.

I thought that we were intending to version library incarnations, and it seems like it would be confusing to not name them differently. What do you think about this change?

Reply:
My rationale is that I modeled the naming after projects like Apache Commons Math. They still call it "Apache Commons Math" as a name, even though the artifact is commons-math4 and the package also is `org.apache.commons.math4`. It's just the name that doesn't have the `4` in it.

Now in the case of SciJava Common, the scope of the library is drastically reduced from what it was before. So anyone thinking that this new scijava-common3 could possibly replace the previous scijava-common is in for a rude shock if they try it. But I'm not super concerned about that.

Anyway, I don't feel super strongly, if you'd rather make the name "SciJava Common 3" then I guess we can do that. Or should we do "SciJava Common3"? I wasn't sure, so I opted to just ditch the "3". :sweat_smile:

Reply:
Gotcha, didn't realize that was the precedent. Let's just go with it then.